css/css-images/gradient/gradient-eval-001-ref.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <style>
+   :root 
```

Comment:
This seems to be a leftover fragment of a selector? In this case it doesn't matter since it chains into the next line and produces `:root .test` as the selector, and that'll always match here, but it's still an extraneous detail.

Reply:
Thanks Tab, both done.